content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst

```diff
@@ -0,0 +1,398 @@
+   specify the :guilabel:`Units` for the lot number. When finished, click :guilabel:`Confirm`.
+
+   .. image:: gs1_nomenclature/edit-lot-quantities.png
+      :align: center
+      :alt: Change scanned quantities using pencil button.
+
+Repeat this process to specify additional lot numbers and quantities in this receipt. Once the
+:guilabel:`Units` are all accounted for, finish the reception by clicking the :guilabel:`Validate`
+button.
+
+.. note::
+   THIS SECTION WILL BE DELETED BEFORE PUBLISHING! It's for the reviewers to test these barcodes
+   easily.
+
+   product: (01)12345678901231
+   qty: (30)00000050
+   Lot#1: (10)LOT0001
+   Lot#2: (10)LOT0002
+
+   .. code-block:: javascript
+
+      // First scan: product
+      odoo.__DEBUG__.services['web.core'].bus.trigger(
+         'barcode_scanned',
+         "0112345678901231",
+         $(".o_web_client")[0],
+      )
+      // Second scan: first Lot#, then enter qty with pencil button
+      odoo.__DEBUG__.services['web.core'].bus.trigger(
+         'barcode_scanned',
+         "10LOT0001",
+         $(".o_web_client")[0],
+      )
+      // Third scan: second Lot#, then enter qty.
+      odoo.__DEBUG__.services['web.core'].bus.trigger(
+         'barcode_scanned',
+         "10LOT0002",
+         $(".o_web_client")[0],
+      )
+
+Alternatively, scan the barcode containing the product, lot number, and quantity to complete the
+receipt operation in fewer steps.
+
+Product + non-unit quantity
+---------------------------
+
+To build a GS1 barcode that contains products measured in a non-unit quantity, like kilograms, for
+example, the following barcode patterns are used:
+
++------------+--------------------------+----------+-------------------+-------------------+
+|    Name    |        Rule Name         |   A.I.   |  Barcode Pattern  |   Field in Odoo   |
++============+==========================+==========+===================+===================+
+| Product    | Global Trade Item Number | 01       | (01)(\\d{14})     | Barcode field on  |
+|            | (GTIN)                   |          |                   | product form      |
++------------+--------------------------+----------+-------------------+-------------------+
+| Quantity in| Variable count of items  | 310[0-5] | (310[0-5])(\\d{6})| Units field on    |
+| kilograms  |                          |          |                   | transfer form     |
++------------+--------------------------+----------+-------------------+-------------------+
+
+Scan barcode on receipt
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To confirm that quantities are correctly interpreted in Odoo, place an order in the *Purchase* app
+using the appropriate unit of measure (:guilabel:`UoM`) for the quantity of products to be
+purchased.
+
+.. seealso::
+   :ref:`Simplify vendor unit conversions with UoMs<inventory/products/uom-example>`
+
+After the order is placed, navigate to the :menuselection:`Barcode` app to receive the :ref:`vendor
```

Comment:
The link is to "Receipts" so I think it makes sense for "receive" to be included in the link name 
```suggestion
After the order is placed, navigate to the :menuselection:`Barcode` app to :ref:`receive the vendor
```